### ISSUE TITLE
better 3C295 model

### ISSUE THREAD

**Issue by rvweeren** [lofar-astron/prefactor#30]
At the moment we are using a two component model for 3C295. I thought this model was sufficiently detailed until I compared the amplitude solutions of this model against the selfcal solutions form factor. The amplitudes behave rather wild for the 3C295 two component model. Does anyone know if the long-baseline people have a better model/image for 3C295  ? (I though I remembered seeing an HBA image at some point made using international baselines…maybe from Adam Deller?)

(I think the flux-scale calibration for the most distant stations, RS310/509/210 is significantly affected, up to 50%, because of the limited model accuracy of the 3C295 model)

**Comment by rvweeren** [comment]
These are the amplitude solutions with the 2 component model of 3C295. Ideally these solutions are flat for each stations (as is the case for 3C196 with the model from Pandey). They are clearly not and this will cause flux-scale calibration problems for the distant RS stations.

![l343226_sbgr025-10_uv dppp pre-cal_chunk9_12656e813t_0g make_selfcal_plots_amp_channel0](https://cloud.githubusercontent.com/assets/3879473/14436034/c9b59e9e-ffe8-11e5-9452-f8c4a75928c7.png).

**Comment by twshimwell** [comment]
Leah has a 8hr international baseline run on 3C295 and is in the process of making a model which could improve this.

**Comment by lmorabit** [comment]
Yes, I am working on this at the moment.  Other people have requested it, and it's high on my to-do list. I have a model at the moment with using only baselines that contain international stations, so you just see the hot spots (which contain about 70% of the total flux). I still need to perform a proper multi-scale clean to recover the extended emission from the lobes.

**Comment by rvweeren** [comment]
I've experimented a bit with the Tier-1 (factor) data that has 3C295 relatively close to the pointing center. Since the source is very bright we should be able to get better resolution than the formal beam size. Attached image is from a first test run and shows the clean component model I obtained after running factor (I hacked the code to decrease the cell size and changed the weighting closer to uniform). This was only using data at 120-126 MHz so further improvement can be expected. 

(this is already significantly better than the 2-component model we are using now)

<img width="715" alt="3c295" src="https://cloud.githubusercontent.com/assets/3879473/14581945/91508f76-03c8-11e6-82bd-be36c9495fef.png">

**Comment by AHorneffer** [comment]
Is there news?
Is there a new skymodel, that we could include in prefactor?

**Comment by rvweeren** [comment]
Not that I know (the long-baseline data is needed to make good model). Leah any update on this ?

**Comment by lmorabit** [comment]
I'm working on this with some long baseline observations, but it probably
won't be ready for a couple of months.

Leah

**Comment by twshimwell** [comment]
Hey @lmorabit. Just wondering if you were still planning on improving the 3c295 model?

**Comment by lmorabit** [comment]
Yes. I've been working with Etienne on this, but I am starting to process
an 8-hour observation of 3C 295 which has the international stations, so
that will also be available within a few weeks to a couple months.

**Comment by twshimwell** [comment]
Great. Any plans for other calibrators that have models for in prefactor (or do the appropriate observations exist?)

**Comment by lmorabit** [comment]
Which calibrators would you like to have?  I have a decent model of 3C 147
already.

**Comment by twshimwell** [comment]
3C 380 might be nice (issue #165). Is your 3C 147 on the Scaife & Heald flux scale? Perhaps we could take a look at the prefactor calibrator plots when running with the old model and the new one.

**Comment by lmorabit** [comment]
Yes, 3C 147 is on the Scaife & Heald flux scale.

**Comment by tikk3r** [comment]
An update on what we have for this. We've used the recent 8 hour scan on 3C295 with all current (13) stations, to try and improve the model, using DPPP+WSClean, but it appears to be a bit difficult getting proper bandpasses from it. The following is what we have obtained so far:
![3C295](https://user-images.githubusercontent.com/2464769/61721562-f0394580-ad68-11e9-98ae-469e76a3752b.png)
This image was made with briggs -2 weighting and an inner cut of 25 klambda. The resolution does look a bit artificial to me (suggestions/ideas welcome), as I don't think we have baselines reaching that resolution in terms of l/b, but it is what WSClean fits and the PSF image does show such a small PSF. A total flux density of 93.86 Jy is recovered (central frequency 144 MHz) from the clean-component model (a bit more than what PyBDSF recovered with standard settings from the wiki). The Astron webpage lists 97.763 Jy at 150 MHz, and the 6C catalog lists ~90 Jy (from NED). I'm not sure what to make of that.

Below are what we get for the bandpasses in three different scenarios (ignore flagged Dutch stations, this was due to hardcoded strict limits during bandpass flagging, flagging all outside 50-250):
**Current two-component model**
![bandpass_rangefix_freq_twocomp_time5064495594 00278](https://user-images.githubusercontent.com/2464769/61721634-11019b00-ad69-11e9-8e61-90ff04d15c8a.png)

**Clean component model from the above image**
![bandpass_rangefix_freq_CC_time5064495594 00278](https://user-images.githubusercontent.com/2464769/61721650-1bbc3000-ad69-11e9-885e-b4ff037460d2.png)

**PyBDSF model from VLA 8 GHz image**
![bandpass_rangefix_freq_VLA_time5064495594 00278](https://user-images.githubusercontent.com/2464769/61721711-37273b00-ad69-11e9-8b30-153d1eaa77b9.png)

**Comment by lmorabit** [comment]
Several suggestions / comments / questions:

1. You're in the right ballpark for the integrated flux density for the source, so I wouldn't worry about the minor discrepancies right now.  You can always go back after the final self-calibration and scale the total flux to a standardised flux scale.

2. For the resolution, can you take a 2D cut across the beam?  I would be interested to see if there are troughs on either side of the beam that is causing the fit to underestimate the width of the beam.  It can be difficult to see this just by looking at the psf image, especially if there is a peak at the centre, surrounded by troughs, with a secondary peak before dropping off to something approaching zero.

3. Are you using the same u-v cuts in prefactor as in the imaging?

4. How many self-cal cycles are you going through with DPPP+wsclean to get the image, and are they mostly phase or are there some phase+amp?  Has the dispersive delay (i.e. the TEC) already been corrected?  The dispersive delay especially is important to get right, before you start self-cal on the phases, otherwise your self-cal solutions may not drive to the right global mininum.

Looking at the image and the solutions, it seems to me that the model is not yet correct. One thing I worry about a lot with such a source is that the spectral indices for all of the components in the image are correct -- we saw this with Etienne's data, where he did self-cal on a block of 10 subbands and then subtracted that model from blocks at other frequencies -- you could see immediately how the spectral index varied across the source. For the standard calibrators, I know it's annoying and time consuming, but I would try the following approach: solve and correct for the TEC to remove the dispersive delay, which allows you to average down in frequency to reduce the field of view, and then use CASA for self-calibration with fringe-fitting and multi-scale, multi-frequency synthesis in clean. (AIPS doesn't have mfs capability)  While you may get decent solutions using DPPP, I would really try to use a fringe-fitting algorithm on these very complex sources where it is crucial that you get every detail right.

**Comment by tikk3r** [comment]
Update after a while. This is currently my best image/model so far (colorbar is clipped at 1000*RMS):
![image](https://user-images.githubusercontent.com/2464769/76311940-4b0cb680-62d2-11ea-9ac1-51c95de37751.png)
which results in the following bandpasses on a calibrator scan that I have lying around:
![bandpass_time5064495594 00278](https://user-images.githubusercontent.com/2464769/76311964-59f36900-62d2-11ea-8af7-ae2975d1607b.png)

**Comment by adrabent** [comment]
The bandpasses for the International Stations look pretty good!
We could include this model in prefactor 3.1 (?)

**Comment by lmorabit** [comment]
What is the difference between prefactor 3.0 and 3.1 ?  I need to base the LOFAR-VLBI pipeline on a stable release.  If it's just minor bug fixes between .0 and .1 then it should be ok -- but I will need to know quite soon as I'm trying to wrap up the publication for the LOFAR-VLBI pipeline.  I have been using the V3.0 tag (as recommended) as the basis for the LOFAR-VLBI pipeline.

**Comment by adrabent** [comment]
No.. prefactor 3.1 or 3.0.1 or whatever will only contain fixes since the 3.0 release and maybe we could think about incorporating high-resolution skymodels for the long baselines.

**Comment by lmorabit** [comment]
Sorry, I'm not sure that answered my question.  What will the difference between 3.0 and 3.1 be, or do you know when you will know what those differences will be?

**Comment by adrabent** [comment]
Only minor fixes which are already present in the master branch .. and maybe a newer model. No major changes. It won't affect the long baseline pipeline.

**Comment by lmorabit** [comment]
I'm not sure that's true, as I had major difficulty with the long baseline pipeline + the master branch, which is why I've reverted to using the V3.0 tag. We need to re-evaluate this before making claims that the master branch will work with the long baseline pipeline.

**Comment by tikk3r** [comment]
As the previous model does not have a fixed flux scale, I also made a version in which the flux density has been rescaled to the Scaife&Heald value and all the spectral indices have been replaced with their Scaife&Heald values. This should result in a better flux scale, but the bandpasses change a little bit. For example, the Irish station gets more wiggles.

**Bandpasses after scaling flux density and replacing spectral indices**
![bandpass_time5064495594 00278](https://user-images.githubusercontent.com/2464769/107951382-8a3e8f80-6f98-11eb-8581-0af5d70314b4.png)

What do people think? Both models are clearly much better than a simple two-component model, so I think it comes down to what is more important: smoother bandpasses or a more or less correct flux scale.
